lighthouse-core/test/results/sample_v2.json

```diff
@@ -209,6 +209,16 @@
       "numericValue": 122.537,
       "displayValue": "120 ms"
     },
+    "cumulative-layout-shift": {
+      "id": "cumulative-layout-shift",
+      "title": "Cumulative Layout Shift",
+      "description": "The more the page's layout changes during its load, the higher the Cumulative Layout Shift. Perfectly solid == 0. Unpleasant experience >= 0.50.",
+      "score": 1,
+      "scoreDisplayMode": "numeric",
+      "numericValue": 0,
+      "displayValue": "0",
+      "explanation": "No LayoutShift trace events found"
```

Comment:
a little weird that the empty case is valid so we have no clue if it didn't work or the page is just good 😆

Reply:
WDYT about my suggestion for connor too that we manually inject a few of these events into our trace to exercise the logic?

Reply:
> a little weird that the empty case is valid so we have no clue if it didn't work or the page is just good 😆

yeah this is kinda rough.

(and for the record the current JSON does include a manually added trace event)


also this:
 https://github.com/GoogleChrome/lighthouse/blob/efb69e66efcc08554e469f8ac5c62aff4ed093aa/lighthouse-core/computed/metrics/cumulative-layout-shift.js#L31-L35

i'm going to change it so we report this state as a success.

Reply:
update: did the above as d09fbb35e 

--------

@tdresser could chrome emit a trace event at ~ window load declaring there's no layoutshift observed?  
otherwise can't distinguish between the case of "chrome that doesn't support this" vs "no layout shifts observed and CLS === 0". 

ofc UA sniffing is an alternative but i'd have more confidence if the signal came straight from the layout shift tracker.